ibm_catalog.json

```diff
@@ -0,0 +1,129 @@
+{
+    "products": [
+        {
+            "name": "deploy-arch-ibm-cos-config",
+            "label": "Object Storage bucket configuration",
+            "product_kind": "solution",
+            "tags": [
+                "ibm_created",
+                "target_terraform",
+                "terraform",
+                "solution",
+                "storage"
+            ],
+            "keywords": [
+                "COS",
+                "bucket",
+                "replication",
+                "object storage",
+                "IaC",
+                "infrastructure as code",
+                "terraform",
+                "solution"
+            ],
+            "short_description": "Creates and configures replication between two regional Object Storage buckets",
+            "long_description": "This sample deployable architecture supports creating an Object Storage instance, a source bucket, a target bucket and a replication rule to replicate all data.",
+            "offering_docs_url": "https://github.com/terraform-ibm-modules/sample-deployable-architectures/blob/main/README.md",
+            "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-cos/main/images/cos_icon.svg",
+            "provider_name": "IBM",
+            "features": [
+                {
+                    "title": "Creates and configures Cloud Object Storage buckets",
+                    "description": "Yes"
```

Comment:
This feature isn't in that variation comparison view, right? I think this appears at the top of the catalog details page as a highlight on the whole solution, so could we put a sentence for the description just to set the example.